Data-Structure/Collection-In-Java/CollectionOfEmployee/NaturalSorting.java

```diff
@@ -0,0 +1,24 @@
+package CollectionOfEmployee;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Niraj Gupta
+ * Class to perfrom natural sorting.
+ *
+ */
+public class NaturalSorting{
+	/**
+	 * Method to perform natural sorting on the list of employees.
+	 * @param employees : The details of the employees.
+	 * @return : The sorted list of employees.
+	 */
+	public static List<Employee> naturalSort(Map<String, Employee> employees){

```

Comment:
on what field of Employee is the sorting applied here?

Reply:
On employee id.